extensions/arc/notebooks/arcDeployment/deploy.postgres.existing.arc.ipynb

```diff
@@ -161,13 +163,18 @@
                 "\n",
                 "workers_option = f' -w {server_group_workers}' if server_group_workers else \"\"\n",
                 "port_option = f' --port \"{server_group_port}\"' if server_group_port else \"\"\n",
+				"engine_version_option = f' --engine-version -ev {server_group_engine_version}' if server_group_engine_version else \"\"\n",
```

Comment:
You can provide either --engine-version or -ev, but not both.  They're the same argument, one is just the short name.